File: ucare_frontend/src/service/timeService.js
import axios from 'axios';
import { TIME_API_BASE_URL } from './urlConfig'

class timeService {
  update(date){
    return axios.put(TIME_API_BASE_URL + '/update/' + date)
  }
  updateByCancel(data){
    return axios.put(TIME_API_BASE_URL + '/updateByCancel', data)
  }
}

export default new timeService();

File: ucare_backend/src/main/java/com/douzone/ucare/service/TimeService.java
package com.douzone.ucare.service;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import com.douzone.ucare.repository.TimeRepository;
import com.douzone.ucare.vo.ReceiptVo;

@Service
public class TimeService {
	
	@Autowired
	private TimeRepository timeRepository;

	public int update(String date) {
		return timeRepository.update(date);
	}

	public int updateByCancel(ReceiptVo data) {
		return timeRepository.updateByCancel(data);
	}
}